packages/xdl/src/Webhooks.ts

```diff
@@ -1,7 +1,15 @@
 import UserManager from './User';
 import ApiV2Client from './ApiV2';
 
-export async function setWebhookAsync(experienceName, { url, secret, event }) {
+export type WebhookEvent = 'build';
+
+export type WebhookData = {
+  url: string;
+  secret: string;
+  event: WebhookEvent;
+};
+
+export async function setWebhookAsync(experienceName: string, { url, secret, event }: WebhookData) {
```

Comment:
add return types -- it's especially helpful here since it requires work to read www's source to figure out what the return type is